eth_abi/registry.py

```diff
@@ -0,0 +1,131 @@
+from cytoolz import curry
+
+from .utils.parsing import collapse_type
+
+
+class ABIRegistryError(Exception):
```

Comment:
We've typically located these in `lib_name.exceptions` by convention.

Reply:
I did notice that and planned to move the exceptions later.  Was just keeping everything in one file while I work on it.  But I'll go ahead and move it there now.